src/components/ui/FunctionDefinition.tsx

```diff
@@ -20,27 +20,26 @@ export const FunctionDefinition: FC<FunctionDefinitionProps> = ({
 	func,
 	prefix = undefined,
 }) => {
+	const funcName = func.self ? "self" : genPath(func);
 	return (
-		<div class="bg-gray-50 p-4 rounded-md border border-gray-100 overflow-x-auto">
-			<div class="code code-definition font-mono text-base">
-				<span class="text-gray-500">{func.self ? "self." : genPath(func)}</span>
-				<span class="typ-func font-semibold text-blue-500">{func.name}</span>
-				<span class="text-gray-700">(</span>
-				<div class="arguments pl-4 md:pl-6 flex flex-col">
+		<pre>
+			<code>
+				<span>{funcName}</span>
+				{funcName !== "" && <span class="typ-punct">.</span>}
```

Comment:
This style assumes that there is no additional `.` in `funcName`. According to the following, it's true at present.

```shell
$ jq '.. | select(type == "object" and has("path")) | .path | length' docs.json | sort | uniq --count
    101 0
    421 1
```

I suggest adding a comment to `genPath.ts`:

> For Typst v0.14.0, the path is either empty or has only one item.